### PR TITLE
[PCFG-56] added tlscert-api-config-dev-platform-pagopa-it

### PR DESCRIPTION
**Pull request by aomegax**
Added tlscert-api-config-dev-platform-pagopa-it

### Type of changes

- [X] Add new pipeline
- [ ] Update pipeline configuration
- [ ] Remove pipeline

### :warning: If it's new pipeline with code review have you added pagopa-github-bot -> Role: admin?

- [ ] Yes
- [X] No

### Other information

---

### CODE REVIEW

File: azure-devops/projects/pagopa-projects/tlscert-api-config-dev-platform-pagopa-it.tf [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  }
+  tlscert-api-config-dev-platform-pagopa-it-variables = {
+    KEY_VAULT_SERVICE_CONNECTION = module.DEV-PAGOPA-TLS-CERT-SERVICE-CONN.service_endpoint_name
+  }
+  tlscert-api-config-dev-platform-pagopa-it-variables_secret = {
+  }
+}
+
+module "tlscert-apiconfig-dev-platform-pagopa-it-cert_az" {
+  source = "git::https://github.com/pagopa/azuredevops-tf-modules.git//azuredevops_build_definition_tls_cert?ref=v2.0.2"
```
Comment (pasqualedevita) [inline azure-devops/projects/pagopa-projects/tlscert-api-config-dev-platform-pagopa-it.tf:41]:
```suggestion
  source = "git::https://github.com/pagopa/azuredevops-tf-modules.git//azuredevops_build_definition_tls_cert?ref=v2.0.1"
```
please use v2.0.1 version, v2.0.2 is in preview and needs others changes (we'll schedule migration on other pr)

File: azure-devops/projects/pagopa-projects/tlscert-api-config-dev-platform-pagopa-it.tf [hunk 2]
```diff
@@ -0,0 +1,74 @@
+variable "tlscert-api-config-dev-platform-pagopa-it" {
```
Comment (pasqualedevita) [inline azure-devops/projects/pagopa-projects/tlscert-api-config-dev-platform-pagopa-it.tf:1]:
```suggestion
variable "tlscert-dev-api-config-dev-platform-pagopa-it" {
```
missing `dev-` prefix in vars and filename

**Review by pasqualedevita — CHANGES_REQUESTED** [review]

**Review by uolter — DISMISSED** [review]

File: azure-devops/projects/pagopa-projects/tlscert-uat-api-config-uat-platform-pagopa-it.tf [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+      name           = "le-azure-acme-tiny"
+      branch_name    = "master"
+      pipelines_path = "."
+    }
+    pipeline = {
+      enable_tls_cert         = true
+      path                    = "TLS-Certificates\\UAT"
+      dns_record_name         = ""
+      dns_zone_name           = "apiconfig.uat.platform.pagopa.it"
+      dns_zone_resource_group = "pagopa-d-vnet-rg"
```
Comment (pasqualedevita) [inline azure-devops/projects/pagopa-projects/tlscert-uat-api-config-uat-platform-pagopa-it.tf:14]:
```suggestion
      dns_zone_resource_group = "pagopa-u-vnet-rg"
```

File: azure-devops/projects/pagopa-projects/tlscert-uat-api-config-uat-platform-pagopa-it.tf [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+      # common variables to all pipelines
+      variables = {
+        CERT_NAME_EXPIRE_SECONDS = "2592000" #30 days
+        KEY_VAULT_NAME           = "pagopa-d-kv"
```
Comment (pasqualedevita) [inline azure-devops/projects/pagopa-projects/tlscert-uat-api-config-uat-platform-pagopa-it.tf:18]:
```suggestion
        KEY_VAULT_NAME           = "pagopa-u-kv"
```

**Review by pasqualedevita — CHANGES_REQUESTED** [review]

**Review by pasqualedevita — APPROVED** [review]

**Comment by pasqualedevita** [comment]
@aomegax next times wait to merge, I need to apply change before merge

**Pull request merged**